scripts/pylint_custom_plugin/pylint_guidelines_checker.py

```diff
@@ -1706,6 +1706,68 @@ def visit_functiondef(self, node):
     visit_asyncfunctiondef = visit_functiondef
 
 
+class CheckNamingMismatch(BaseChecker):
+    __implements__ = IAstroidChecker
+
+    name = "check-naming"
+    priority = -1
+    msgs = {
+        "C4745": (
+            "Exposing aliased generated code."
```

Comment:
Does this line need to tell people NOT to expose aliased generated code?

Reply:
I changed it to say do not alias generated code